BreedCheckbox/BreedCheckbox.tsx

```diff
@@ -6,14 +6,16 @@ import FormControl from "@material-ui/core/FormControl";
 
 type Props = {
   breed: string;
+  handleCheckbox: (option: string) => void;
```

Comment:
handleCheck, handleChecking, onCheck, or even just `onChange` would be much more better